src/UI/Component/Entity/Entity.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\UI\Component\Entity;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Image\Image;
+use ILIAS\UI\Component\Symbol\Symbol;
+use ILIAS\UI\Component\Symbol\Glyph\Glyph;
+use ILIAS\UI\Component\Button\Shy;
+use ILIAS\UI\Component\Button\Tag;
+use ILIAS\UI\Component\Legacy\Legacy;
+use ILIAS\UI\Component\Listing\Property as PropertyListing;
+use ILIAS\UI\Component\Link\Standard as StandardLink;
+
+/**
+ * This describes an Entity
+ */
+interface Entity extends Component
+{
+    //Priority Areas
+
+    /**
+     * Set (and maybe explain) Blocking Availability Conditions when there is
+     * some access restriction for the user on this object.
+     *
+     * @param string|PropertyListing|StandardLink|Legacy|array<PropertyListing|StandardLink|Legacy> $blocking_conditions
```

Comment:
To simplify this typehint:

* lets drop `PropertyListing|StandardLink|Legacy` because these also could be passed via an array with one element
* lets drop `string` because we don't know much about what would be contained anyway and could just use `Legacy` instead

We could then hint like:

```suggestion
     PropertyListing|StandardLink|Legacy ...$blocking_conditions
```

and drop `@param` in the doc-block completely.